lib/pact.ex

```diff
@@ -110,6 +120,21 @@ defmodule Pact do
     {:stop, :normal, :ok, container}
   end
 
+  defp create_fake_module_name(name, pid) do
+    process_id_as_string = IO.iodata_to_binary(:erlang.pid_to_list(pid))
+    module_name = String.capitalize(name)
+    String.to_atom("PactFakes#{module_name}#{process_id_as_string}")
+  end
+
+  defp stub_module(module, overridden_functions) do
+    :meck.new(module, [:no_link, :non_strict])
```

Comment:
It's not much more code, and it saves us a dependency so I lean towards yes. Your thoughts?